auto_round/mllm/autoround_mllm.py

```diff
@@ -24,27 +24,47 @@
     to_dtype,
     get_multimodal_block_names,
     find_matching_blocks,
-    extract_block_names_to_str
+    extract_block_names_to_str,
+    clear_memory
 )
 from ..autoround import AutoRound
-from .template import get_template, Template
+from .template import get_template, Template, SUPPORT_ONLY_TEXT_MODELS
 from .mllm_dataset import get_mllm_dataloader
 from ..low_cpu_mem.utils import get_layers_before_block
 
 
-def _only_text_test(model, tokenizer, device):
+def _only_text_test(model, tokenizer, device, model_type):
     """Test if the model whether can use text-only datasets."""
+
+    if model_type in SUPPORT_ONLY_TEXT_MODELS:  # save time
+        return True
+
+    device = detect_device(device)
+    text = ["only text", "test"]
+    tokenizer.padding_side = 'left'
```

Comment:
it tokenizer has been changed here, the saved tokenizer should be changed too, right? That's not the expected behavior, the saved tokenizer should be exact the same as the original one.

Better revert the change after this thest